src/main/java/core/basesyntax/service/impl/OperationValidatorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exception.UnknownOperationException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationValidator;
+
+public class OperationValidatorImpl implements OperationValidator {

```

Comment:
let's remove this interface and its implementation and create a separate method with name findOperationByFirstLetter with the same logic inside of enum FruitOperation